### PR TITLE
Docs(bitmapSubsetLimit): Added bitmapSubsetLimit description

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/query_language/functions/bitmap_functions.md [hunk 1]
```diff
@@ -82,30 +82,42 @@ SELECT bitmapToArray(bitmapSubsetInRange(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,
 └───────────────────┘
 ```
 
-## bitmapSubsetLimit {#bitmap_functions-bitmapsubsetlimit}
+## bitmapSubsetLimit {#bitmapsubsetlimit}
 
-Return subset of the smallest `limit` values in set which is no less than `range_start`.
+Returns a subset of a bitmap. The subset values are sorted in ascending order and its cardinality and starting point do not exceed the corresponding input values.
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/bitmap_functions.md]: Don't understand.

File: docs/ru/query_language/functions/bitmap_functions.md [hunk 2]
```diff
@@ -48,6 +48,44 @@ SELECT bitmapToArray(bitmapBuild([1, 2, 3, 4, 5])) AS res
 └─────────────┘
 ```
 
+## bitmapSubsetLimit {#bitmapsubsetlimit}
+
+Возвращает подмножество битового массива. Значения подмножества сортируются в порядке возрастания, а его верхний предел и начальная точка не превышают соответствующие входные значения.
```
Comment (alexey-milovidov) [inline docs/ru/query_language/functions/bitmap_functions.md]: That's a bitmap, not a bit array.

Comment (alexey-milovidov) [inline docs/ru/query_language/functions/bitmap_functions.md]: Don't understand the logic of this function.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
This documentation does not make sense for me.

**Comment by godfreyd** [comment]
Fixed

**Comment by alexey-milovidov** [comment]
Ok.

**Pull request merged**